x-pack/legacy/plugins/lens/public/indexpattern_plugin/operation_definitions/terms.tsx

```diff
@@ -57,7 +62,10 @@ export const termsOperation: OperationDefinition<TermsIndexPatternColumn> = {
       isBucketed: true,
       params: {
         size: 5,
-        orderBy: { type: 'alphabetical' },
+        orderBy: existingMetricColumn
+          ? { type: 'column', columnId: existingMetricColumn }
+          : { type: 'alphabetical' },
```

Comment:
I left this comment in a couple places but I think we need a special "count" sort order which is the natural sort order of Terms, without referencing a separate column.